improver_tests/calibration/ensemble_calibration/test_ContinuousRankedProbabilityScoreMinimisers.py

```diff
@@ -32,12 +32,12 @@
 Unit tests for the
 `ensemble_calibration.ContinuousRankedProbabilityScoreMinimisers`
 class.
-
 """
 import unittest
 
 import iris
 import numpy as np
+from iris.cube import CubeList
```

Comment:
I think we need 4 new unit tests here, probably for the top level process method.
* Truncated normal with global coeffs and additional predictor
* Normal with global coeffs and additional predictor
* Truncated normal with site specific coeffs and additional predictor
* Normal with site-specific coeffs and additional predictor

Reply:
As agreed, I've implemented tests for the following:
- Normal distribution, "global" coefficients, sites, using additional predictor
- Normal distribution, point by point coefficients, sites, using additional predictor
- Truncated normal, "global" coefficients, gridded, using additional predictor